src/main.js

```diff
@@ -5,8 +5,13 @@ import { openSwingStore } from '@agoric/swing-store-simple';
 // import { runKeybaseBot } from './keybase';
 import { runDiscordBot } from './discord';
 
-const DEFAULT_NETWORK_NAME = 'testnet';
-const VALID_NETWORK_NAMES = [DEFAULT_NETWORK_NAME, 'hacktheorb'];
+const DEFAULT_NETWORK_NAME = 'devnet';
+const INCENTIVIZED_TESTNET_NETWORK_NAME = 'intest';
+const VALID_NETWORK_NAMES = [
+  DEFAULT_NETWORK_NAME,
+  INCENTIVIZED_TESTNET_NETWORK_NAME,
```

Comment:
You may want to rename `VALID_NETWORK_NAMES` to `SPECIFIABLE_NETWORK_NAMES`, and then remove `INCENTIVIZED_TESTNET_NETWORK_NAME` from it, as without that people can do:

```
!faucet testnet delegete ...
```

even when on the regular #faucet channel.

I've updated the deployment server to accept `devnet`.  That should work anytime now.